[0.018s] Invoking command in '/workspace/build/action_tutorials_interfaces': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/workspace/install/srvcli:/workspace/install/pubsub:/workspace/install/my_py_pkg:/workspace/install/my_package:/opt/ros/humble /usr/bin/cmake --build /workspace/build/action_tutorials_interfaces -- -j12 -l12
[0.048s] -- Found ament_cmake: 1.3.3 (/opt/ros/humble/share/ament_cmake/cmake)
[0.186s] -- Found ament_lint_auto: 0.12.5 (/opt/ros/humble/share/ament_lint_auto/cmake)
[0.249s] -- Found rosidl_default_generators: 1.2.0 (/opt/ros/humble/share/rosidl_default_generators/cmake)
[0.251s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.254s] -- Found rosidl_adapter: 3.1.4 (/opt/ros/humble/share/rosidl_adapter/cmake)
[0.281s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.403s] [31mCMake Error at /opt/ros/humble/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:59 (message):
[0.403s]   execute_process(/usr/bin/python3.10 -m rosidl_adapter --package-name
[0.403s]   action_tutorials_interfaces --arguments-file
[0.403s]   /workspace/build/action_tutorials_interfaces/rosidl_adapter__arguments__action_tutorials_interfaces.json
[0.403s]   --output-dir
[0.403s]   /workspace/build/action_tutorials_interfaces/rosidl_adapter/action_tutorials_interfaces
[0.403s]   --output-file
[0.403s]   /workspace/build/action_tutorials_interfaces/rosidl_adapter/action_tutorials_interfaces.idls)
[0.403s]   returned error code 1:
[0.403s] 
[0.403s]   Error processing 'int31 success' of
[0.403s]   'action_tutorials_interfaces/Fibonacci_Feedback': ''int31' is an invalid
[0.403s]   message name.  It should have the pattern '^[A-Z][A-Za-z0-9]*$''
[0.404s] 
[0.404s]   Traceback (most recent call last):
[0.404s] 
[0.404s]     File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
[0.404s]       return _run_code(code, main_globals, None,
[0.404s]     File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
[0.404s]       exec(code, run_globals)
[0.404s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__main__.py", line 19, in <module>
[0.404s]       sys.exit(main())
[0.404s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/main.py", line 53, in main
[0.404s]       abs_idl_file = convert_to_idl(
[0.405s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__init__.py", line 29, in convert_to_idl
[0.405s]       return convert_action_to_idl(
[0.405s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/action/__init__.py", line 28, in convert_action_to_idl
[0.405s]       action = parse_action_string(package_name, input_file.stem, content)
[0.405s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 906, in parse_action_string
[0.405s]       feedback_message = parse_message_string(
[0.405s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 520, in parse_message_string
[0.405s]       Type(type_string, context_package_name=pkg_name),
[0.405s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 277, in __init__
[0.405s]       super(Type, self).__init__(
[0.405s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 205, in __init__
[0.405s]       raise InvalidResourceName(
[0.405s] 
[0.405s]   rosidl_adapter.parser.InvalidResourceName: 'int31' is an invalid message
[0.406s]   name.  It should have the pattern '^[A-Z][A-Za-z0-9]*$'
[0.406s] 
[0.406s] Call Stack (most recent call first):
[0.406s]   /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[0.406s]   CMakeLists.txt:27 (rosidl_generate_interfaces)
[0.406s] 
[0.406s] [0m
[0.406s] -- Configuring incomplete, errors occurred!
[0.406s] See also "/workspace/build/action_tutorials_interfaces/CMakeFiles/CMakeOutput.log".
[0.410s] gmake: *** [Makefile:747: cmake_check_build_system] Error 1
[0.413s] Invoked command in '/workspace/build/action_tutorials_interfaces' returned '2': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/workspace/install/srvcli:/workspace/install/pubsub:/workspace/install/my_py_pkg:/workspace/install/my_package:/opt/ros/humble /usr/bin/cmake --build /workspace/build/action_tutorials_interfaces -- -j12 -l12
